### PR TITLE
Add ability to change stripe account

### PR DESCRIPTION
**Pull request by mortimr**
Hi, I'm willing to add a new functionality to this project. We currently cannot set the stripe account id on iOS, and wanted to create a method to do it. I managed to make this quick "fix" but I'm willing to add it to work properly on web and android aswell. It is a very important feature for Stripe Connect users

### CODE REVIEW

**Comment by ihadeed** [comment]
There are examples on how to do it on web + Android here: https://stripe.com/docs/connect/authentication#adding-the-connected-account-id-to-a-client-side-application

I think the implementation should return a new instance of the client for the provided account id, or have another function to reset it.

File: packages/plugin/ios/Plugin/Plugin.swift [hunk 1]
```diff
@@ -14,6 +14,19 @@ public class StripePlugin: CAPPlugin {
     
     internal var ERR_NO_ACTIVE_CUSTOMER_CTX = "No active customer session was found. You must crete one by calling initCustomerSession"
 
+    @objc func setStripeAccount(_ call: CAPPluginCall) {
+        let value = call.getString("strip_account") ?? ""
```
Comment (ihadeed) [inline packages/plugin/ios/Plugin/Plugin.swift:18]: Typo in option name
`strip_account` -> `stripe_account`

**Comment by mortimr** [comment]
Ok, I'll try to do something to create a new instance of the SDK instead of what is currently done. I have some work to finish first. Thanks for your reply !

**Comment by timsayshey** [comment]
@mortimr Thanks for tackling this -- Any updates?

**Comment by LinnaeK** [comment]
Will this, or another built in solution for adding sub accounts for Stripe via the plugin be available in the near future?

**Comment by mortimr** [comment]
Hi ! I'm currently using my own fork for this. I have a pretty filled schedule and I am not going to be able to implement and add it properly in this repo until January. I'll try my best to get back to this and do it. Sorry for the time it takes

**Comment by LinnaeK** [comment]
Hi @mortimr! Just checking in to see if any progress has been made on including subAccounts with this plugin. Thanks!

**Comment by timsayshey** [comment]
@mortimr Can you comment with a link to the fork you created?

**Comment by rdlabo** [comment]
I'm rdlabo, and I've joined the maintainer of this plugin. Thanks for pull request.
This pull request will be released in v3. Thank you for your contribution.

**Pull request merged**